apps/glusterfs/app_config.go

```diff
@@ -43,10 +43,15 @@ type GlusterFSConfig struct {
 
 	// server behaviors
 	IgnoreStaleOperations          bool   `json:"ignore_stale_operations"`
+	DisableMonitorGlusterNodes     bool   `json:"disable_monitor_gluster_nodes"`
```

Comment:
has this sneaked in accidentally here?

Reply:
ok. i see, the next PR treats disabling/enabling both background tasks, cleanup and node monitor